testdata/serviceconfig.yml

```diff
@@ -30,7 +30,17 @@ rules:
   - kind: access-logs
     params:
       logName: "access_log"
-      logFormat: 0 # Common Log Format
+      log:
+        log_format: 1 # apache common log format
+        template_expressions:
```

Comment:
I think ultimately the common/combined log templates should not be hard coded but migrated to descriptors. That way their config has both pieces.